### DISCUSSION TITLE
Optane.Client project doesn't load in Visual Studio

### DISCUSSION THREAD

**Discussion by ghost** [oqtane/oqtane.framework#854] [Q&A]
I cloned the repo and opened it in Visual Studio 2019. The client project doesn't load. I get this error:
 The project file cannot be opened. C:\Program Files\dotnet\sdk\3.1.403\Sdks\Microsoft.NET.Sdk.BlazorWebAssembly\Sdk not found. Check that a recent enough .NET Core SDK is installed and/or increase the version specified in global.json.

I checked the path and I don't have the Microsoft.NET.Sdk.BlazorWebAssembly folder.
I installed these files:
dotnet-sdk-3.1.403-win-x64.exe
aspnetcore-runtime-3.1.9-win-x64.exe
dotnet-runtime-3.1.9-win-x64.exe
dotnet-hosting-3.1.9-win.exe
windowsdesktop-runtime-3.1.9-win-x64.exe

Plus  SDK & ASP.NET Core 5.0 RC2.

So what am I still missing to install?

**Comment by hishamco** [comment]
Ya, this is happened to me after upgrade to .NET 5 RC1 & RC2, basically what you need is to install the latest VS 2019, or use VS Code like what I'm doing ;)

**Reply by hishamco** [reply]
More precisely Visual Studio 2019 (v16.8, Preview 4)

**Reply by sbwalker** [reply]
I added this VS version to the install instructions:  https://github.com/oqtane/oqtane.framework/blob/master/README.md

**Reply by ghost** [reply]
I installed the VS preview and it's working now. Thanks.

**Comment by markdav-is** [comment]
@thenrich2009  the other tip is to pin yourself to the latest released version.  In this case 1.0.4 https://github.com/oqtane/oqtane.framework/releases/tag/v1.0.4 that is compatible with the latest stable release of VS and dotnet.  We do that with our team to cut down on installing extra preview stuff.  Our team is not contributing back to the project yet, so we don't need to be on the latest development effort.  New version should be out in a few weeks.

**Reply by ghost** [reply]
Where in this tag or web page does it say which version of VS it's compatible with?

**Reply by markdav-is** [reply]
it's not called out in the tag.  I just know that the head of the repo is new work on the next release, and 1.0.4 is the last stable release.
